pkg/kube/kube_handler.go

```diff
@@ -421,20 +421,6 @@ type appRepositoryRequestDetails struct {
 	PassCredentials       bool                    `json:"passCredentials"`
 }
 
-type HttpValidator interface {
-	IsValid(cli httpclient.Client) (*ValidationResponse, error)
-	getReq() *http.Request
-	getAppRepo() *v1alpha1.AppRepository
-}
-
-type HelmNonOCIValidator struct {
-	req *http.Request
-}
-
-type HelmOCIValidator struct {
-	appRepo *v1alpha1.AppRepository
-}
-
```

Comment:
I've just moved the interface and related structs down to where the implementation is.